experimental/dds/tree2/src/test/feature-libraries/chunked-forest/codec/checkEncode.ts

```diff
@@ -82,6 +83,12 @@ function testDecode(
 	const result = decode(chunk);
 	assertChunkCursorEquals(result, tree);
 
+	// handles can't be roundtripped through JSON. avoid the below roundtripping
```

Comment:
Rather than this, can you do something like

```
// defined at top of file
const serializer = new FluidSerializer(new MockHandleContext(), (handle: IFluidHandle) => {});

// in function
const json = serializer.stringify(chunk);
const parsed = serializer.pars(json);
```

Reply:
I tried that, but roundtripping through the serializer converts the MockHandle to a proper handle. Since we compare the round-tripped payload to the original object, the comparison fails. I though about implementing some sort of special comparison function that ignores handles, but that seemed excessive and I'm not sure this JSON-roundtripping property is even one we want to guarantee.

Reply:
that's fair enough. I'd leave a comment that round-tripping with `FluidSerializer` doesn't work either since it round-trips a mock handle into a real one, then.